src/lexoRank/lexoRank.ts

```diff
@@ -311,6 +311,69 @@ export class LexoRank {
     return new LexoRank(this.bucket, LexoRank.between(this.decimal, other.decimal));
   }
 
+  public betweenLexoRanks(other: LexoRank, ranksToGenerate: number): LexoRank[] {
+
+    if (!this.bucket.equals(other.bucket)) {
+      throw new Error('Between works only within the same bucket');
+    }
+
+    if (ranksToGenerate == 0) {
+      return [];
+    }
+
+    let newLeft: LexoRank = this;
+    let newRight: LexoRank = other;
+
+    const cmp = this.decimal.compareTo(other.decimal);
+    if (cmp > 0) {
+      newLeft = other;
+      newRight = this;
+    }
+
+    if (cmp === 0) {
+      throw new Error(
+          'Try to rank between issues with same rank this=' +
+          this +
+          ' other=' +
+          other +
+          ' this.decimal=' +
+          this.decimal +
+          ' other.decimal=' +
+          other.decimal
+      );
+    }
+
+    if (ranksToGenerate == 1) {
```

Comment:
Please move this test immediately below the test for `ranksToGenerate == 0`, and change the implementation to just delegate to the existing `between()` method.

Reply:
Moved this logic after `ranksToGenerate == 0` and used existing method.